debian/control

```diff
@@ -12,6 +12,8 @@ Build-Depends:
   libxkbcommon-dev,
   libwayland-dev,
   pkg-config,
+Depends:
```

Comment:
This needs to move down to the depends below that include shlibs.

Reply:
Should be good now.